src/Scaffold_Command.php

```diff
@@ -201,6 +200,106 @@ private function _scaffold( $slug, $assoc_args, $defaults, $subdir, $templates )
 		}
 	}
 
+	/**
+	 * Generate PHP, JS and CSS code for registering a custom block.
+	 *
+	 * ## OPTIONS
+	 *
+	 * <slug>
+	 * : The internal name of the block.
+	 *
+	 * [--title=<title>]
+	 * : The display title for your block.
+	 *
+	 * [--dashicon=<dashicon>]
+	 * : The dashicon to make it easier to identify your block.
+	 *
+	 * [--category=<category>]
+	 * : The category name to help users browse and discover your block.
```

Comment:
Could/should this category impact scaffolded code?

Reply:
Yes, it is consumed here: https://github.com/wp-cli/scaffold-command/pull/96/files#diff-5384a491183730a8d9db2944d7fcb8d6R13.